source/man/doveadm-mailbox.1.rst.in

```diff
@@ -109,6 +109,68 @@ already deleted.
 **-8**
    Lists the mailboxes with UTF-8 encoding.
 
+mailbox metadata list
+---------------------
+
+**mailbox metadata list** [**-A** | **-u** *user* | **-F** *file*] [**-S** *socket_path*] [**-s**] [**-p**] *mailbox* [*key-prefix*]
+
+**-s**
+   Handle "server metadata" rather than "mailbox matadata".
+
+**-p**
+   Prepend prefix.
+
+*mailbox*
+   The target mailbox.
+
+*key-prefix*
+   The key prefix to look for. All keys will be listed if not provided.
+
+mailbox metadata get
+--------------------
+
+**mailbox metadata get** [**-A** | **-u** *user* | **-F** *file*] [**-S** *socket_path*] [**-s**] *mailbox* *key*
+
+**-s**
+   Handle "server metadata" rather than "mailbox matadata".
+
+*mailbox*
+   The target mailbox.
+
+*key*
+   The key to retrieve.
+
+mailbox metadata set
+--------------------
+
+**mailbox metadata set** [**-A** | **-u** *user* | **-F** *file*] [**-S** *socket_path*] [**-s**] *mailbox* *key* *value*
+
+**-s**
+   Handle "server metadata" rather than "mailbox matadata".
+
+*mailbox*
+   The target mailbox.
+
+*key*
+   The key to set.
+
+*value*
+   The value to set.
+
+mailbox metadata unset
+----------------------
+
+**mailbox metadata unset** [**-A** | **-u** *user* | **-F** *file*] [**-S** *socket_path*] [**-s**] *mailbox* *key*
+
+**-s**
+   Handle "server metadata" rather than "mailbox matadata".
```

Comment:
Wonder if the mailbox name must be `""` or should be.

Reply:
the switch just enables to input "". If the switch is asserted and the mailbox is something else, it ignores the switch